tests/console/unzip.pm

```diff
@@ -24,30 +24,31 @@ use testapi;
 sub run {
     # Preparation
     select_console 'root-console';
-    assert_script_run 'mkdir -p /tmp/unzip-test/';
+    assert_script_run 'mkdir -p /tmp/unzip-test/ && pushd /tmp/unzip-test';
 
     # 1. Unzip (basic usage)
-    assert_script_run 'wget --quiet ' . data_url('console/test_unzip.zip') . ' -O /tmp/unzip-test/archive.zip';
-    assert_script_run 'cd /tmp/unzip-test; unzip archive.zip';
+    assert_script_run 'wget --quiet ' . data_url('console/test_unzip.zip');
+    assert_script_run 'unzip test_unzip.zip';
 
-    my $entries = script_output('ls -1Uq /tmp/unzip-test/ | wc -l');
-    die "Extract produced too few values: $entries instead of 6" unless ($entries =~ /^6$/);
+    my $entries = 'entries1st=' . script_output('ls -1Uq /tmp/unzip-test/ | wc -l');
+    die "Extract produced too few values: $entries instead of 6" unless ($entries =~ /entries1st=6$/);
 
     # Go one step further and verify the md5sum of each file
-    assert_script_run 'wget --quiet ' . data_url('console/checklist.md5') . ' -O /tmp/unzip-test/checklist.md5';
-    assert_script_run 'md5sum -c /tmp/unzip-test/checklist.md5';
+    assert_script_run 'wget --quiet ' . data_url('console/checklist.md5');
+    assert_script_run 'md5sum -c checklist.md5';
 
     # 2. Unzip into a new directory
-    assert_script_run 'cd /tmp/unzip-test/ ; unzip -d extract/ /tmp/unzip-test/archive.zip';
-    $entries = script_output('ls -1Uq /tmp/unzip-test/extract/ | wc -l');
-    die "Extract produced too few values: $entries instead of 5" unless ($entries =~ /^5$/);
+    assert_script_run 'unzip -d extract/ test_unzip.zip';
+    $entries = 'entries2nd=' . script_output('ls -1Uq extract/ | wc -l');
+    die "Extract produced too few values: $entries instead of 5" unless ($entries =~ /entries2nd=5$/);
     # Go one step further and verify the md5sum in extract folder of each file.
-    assert_script_run 'cp /tmp/unzip-test/archive.zip /tmp/unzip-test/extract/archive.zip';
-    assert_script_run "sed -Ei 'sXunzip-testXunzip-test/extractXg' /tmp/unzip-test/checklist.md5";
-    assert_script_run 'md5sum -c /tmp/unzip-test/checklist.md5';
+    assert_script_run 'cp test_unzip.zip extract/test_unzip.zip';
+    #assert_script_run "sed -Ei 'sXunzip-testXunzip-test/extractXg' checklist.md5";
```

Comment:
I guess this line shouldn't be commented out. Otherwise let's remove it =)

Reply:
The line was not necessary. Re-verified.